src/audio/eq_fir/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
 # SPDX-License-Identifier: BSD-3-Clause
 
-add_local_sources(sof eq_fir.c eq_fir_generic.c eq_fir_hifi2ep.c eq_fir_hifi3.c)
+add_local_sources(sof ../module_adapter/module_adapter.c ../module_adapter/module/generic.c eq_fir.c eq_fir_generic.c eq_fir_hifi2ep.c eq_fir_hifi3.c)
```

Comment:
Does this duplicate module adapter source to each component?

Reply:
thanks